volatility3/framework/plugins/yarascan.py

```diff
@@ -119,40 +184,26 @@ def get_yarascan_option_requirements(
             ),
         ]
 
-    @classmethod
-    def yara_returns_instances(cls) -> bool:
```

Comment:
This is a classmethod that may be used in other plugins (which is why it was pulled out as a separate function in the first place), please avoid changing things without good reason.

Reply:
Removing this, in itself, would be reason to bump the version number to `(2, 0, 0)` let alone adding yara-x support.